lib/main_containers.pm

```diff
@@ -80,7 +80,12 @@ sub load_host_tests_podman {
         load_3rd_party_image_test('podman');
         loadtest 'containers/podman_firewall';
         loadtest 'containers/buildah' unless is_sle_micro;
-        loadtest 'containers/rootless_podman' unless is_sle('=15-sp1');    # https://github.com/containers/podman/issues/5732#issuecomment-610222293
+        # https://github.com/containers/podman/issues/5732#issuecomment-610222293
+        unless (is_sle('=15-sp1')) {
+            loadtest 'containers/prepare_rootless_podman';
+            loadtest 'boot/boot_to_desktop';
```

Comment:
Why do we need `boot_to_desktop` a second time? Shouldn't the system just do a normal reboot and boot again into the system?

Reply:
So I found `$self->wait_boot()` so you were right.